### PR TITLE
Some new exercises and tests

### PR DESCRIPTION
**Pull request by shhmon**
Not sure if some are too hard or too easy.

### CODE REVIEW

**Comment by shhmon** [comment]
What I ment with the update of _virtual environment creation_ description was that Windows users won't need to add the pyvenv script to their PATH system variable. Nor will Linux users have to download the pyvenv package when it's already provided by Python 3.X. This method of creating virtual environments works with all OS:es as far as I know, so why not use it.

File: tests/test_introduction.py [hunk 1]
```diff
@@ -1,6 +1,46 @@
...
+    assert i.travel_price(105.5, 0.8, 14) == 118.16
+
+def test_palindrome():
+    assert i.palindrome('ni talar bra latin') == True
+    assert i.palindrome('A car a man a maraca') == True
+    assert i.palindrome('hello there') == False
+    
+def test_character_frequency():
+    assert i.character_frequency('hej janne') == {'h' : 1, 'e' : 2, 'j' : 2, 'a' : 1, 'n' : 2}
+    assert i.character_frequency('bananer') = {'b' : 1, 'a' : 2, 'n' : 2, 'e' : 1, 'r' : 1}
```
Comment (Greenheart) [inline tests/test_introduction.py]: Line 46 is causing the [failing Travis-build](https://travis-ci.org/teknik-eksjo/introduktion/builds/115848297).

Add another '='

**Comment by Limpan** [comment]
https://github.com/Catagen/introduktion/commit/ec6d82b7633e3f4a5dbe93cbf0ef87352e4b8fe0 It's a clear improvement but for me it doesn't work when the default is Python 2.

**Comment by shhmon** [comment]
@Limpan When Python 2 is the default version on for example Ubuntu, you would have to call _python3 -m venv venv_ or replace _python3_ with whatever keyword that runs Python 3 on your system. It does not work with Python 2 since pyvenv is not included in those distributions as you probably know.

**Comment by Limpan** [comment]
@Catagen Yep. I'll write something up.

**Pull request closed without merging**